enterprise/internal/batches/store/changeset_specs.go

```diff
@@ -173,6 +175,35 @@ func (s *Store) UpdateChangesetSpecBatchSpecID(ctx context.Context, cs []int64,
 	return s.Exec(ctx, q)
 }
 
+// UpdateChangesetSpecCommitVerification updates the given ChangesetSpecs CommitVerification if commit signing is applied and successful.
+func (s *Store) UpdateChangesetSpecCommitVerification(ctx context.Context, batchSpec int64, commit *github.RestCommit) (err error) {
+	ctx, _, endObservation := s.operations.updateChangesetSpecCommitVerification.With(ctx, &err, observation.Args{Attrs: []attribute.KeyValue{
+		// TODO: add commit verification status
+		// attribute.Int("Count", len(cs)),
+	}})
+	defer endObservation(1, observation.Args{})
+
+	cv, err := jsonbColumn(commit.Verification)
+
+	if err != nil {
+		return err
+	}
+
+	if commit.Verification.Verified == false {
+		cv, err = jsonbColumn(nil)
+	}
```

Comment:
Maybe just to clean this up a tad and make sure we handle the second `err` if there was any:

```suggestion
	var cv json.RawMessage
	if commit.Verification.Verified {
		cv, err = jsonbColumn(commit.Verification)
	} else {
		cv, err = jsonbColumn(nil)
	}

	if err != nil {
		return err
	}
```

Reply:
Ah, thanks! This is better 😄